.gitignore

```diff
@@ -3,3 +3,8 @@
 /telegraf.exe
 /telegraf.gz
 /vendor
+plugins/inputs/.DS_Store
+plugins/.DS_Store
+OpenWrt.sparseimage
+telegraf.arm64
+.DS_Store
```

Comment:
Revert this file, though I suggest adding .DS_Store to your systemwide ignore file, if you don't have one you can add it with:
```
git config --global core.excludesfile $HOME/.gitignore
```

Then add `.DS_Store` to `$HOME/.gitignore`.

Reply:
Done.